opentelemetry-sdk/tests/trace/test_trace.py

```diff
@@ -114,6 +114,34 @@ def test_start_span_explicit(self):
             self.assertIs(tracer.get_current_span(), root)
             self.assertIsNotNone(child.end_time)
 
+    def test_span_members(self):
```

Comment:
Does this test check anything? Like if the attribute was overwritten properly?

Reply:
No, it does not test. I think it can be added later when the attributes are actually used (when serialising?). I will add a comment in the commitmsg though.